tensorflow/tools/ci_build/install/install_python38.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+cd /usr/src
+wget https://www.python.org/ftp/python/3.8.2/Python-3.8.2.tgz
+tar xzf Python-3.8.2.tgz
```

Comment:
From Google's container tensorflow/tensorflow:devel, they're using Python 3.8.10, do we want to install the same version as they have just for consistence?

Reply:
Ok, I will update to 3.8.10 and re-test for a while.

Reply:
The tests with 3.8.10 look fine. https://tensorflow-ci.intel.com/view/Monitor%20These/job/tensorflow-oob-linux-cpu/2296/consoleFull